apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -80,15 +80,22 @@ export const ALL_PARTNERS_VALUE = "all";
 export const UNMATCHED_PARTNER_LABEL = "No Partner/Unmatched";
 export const UNMATCHED_PARTNER_VALUE = "none";
 
-export const ALL_PARTNERS_OPTION = {label: ALL_PARTNERS_LABEL, value: ALL_PARTNERS_VALUE};
-export const UNMATCHED_PARTNER_OPTION = {label: UNMATCHED_PARTNER_LABEL, value: UNMATCHED_PARTNER_VALUE};
+export const WorkshopTypes = {
+  TEACHERCON: "teachercon",
+  LOCAL_SUMMER: "local summer",
+  BOTH: "both"
+};
+
+export const ALL_PARTNERS_OPTION = {label: ALL_PARTNERS_LABEL, value: ALL_PARTNERS_VALUE, workshopType: WorkshopTypes.BOTH};
+export const UNMATCHED_PARTNER_OPTION = {label: UNMATCHED_PARTNER_LABEL, value: UNMATCHED_PARTNER_VALUE, workshopType: WorkshopTypes.BOTH};
```

Comment:
Are these `workshopType` fields used anywhere?

Reply:
nope, good catch - I just check for the `UNMATCHED_PARTNER_VALUE` or `ALL_PARTNERS_VALUE` so I don't need the workshop type stored here.